local_test_env.yml

```diff
@@ -0,0 +1,15 @@
+# This file is used to install packages for local testing
```

Comment:
I think the name of this file isn't super clear. Let's call it `shared_python_ci_env.yml` and have this comment explain this is a shared test env between all examples for python unit testing

Reply:
Agree, applied